R/join_keys-c.R

```diff
@@ -28,13 +28,13 @@ c.join_keys <- function(...) {
     f = function(.x, .y) {
       assert_compatible_keys2(.x, .y)
       out <- utils::modifyList(.x, .y, keep.null = FALSE)
-      attr(out, "__parents__") <- .merge_parents(.x, .y)
+      attr(out, "__parents__") <- .merge_parents(.x, .y) # nolint
```

Comment:
Is `#nolint` is for the double floor or the period in function name?

We should clean up deviations from our style guide at some point.

Reply:
It's for the "\_\_parents\_\_" name. We can also replace the attribute name to "parents" without underscores

```suggestion
      attr(out, "__parents__") <- .merge_parents(.x, .y) # nolint: object_name_linter.
```